cluster/common.sh

```diff
@@ -299,6 +299,7 @@ function set_binary_version() {
 function find-tar() {
   local -r tarball=$1
   locations=(
+    "${KUBE_ROOT}/node/${tarball}"
```

Comment:
cause I couldn't remember: this path is coming from https://github.com/kubernetes/kubernetes/blob/5c780c8cb134bad1439e170c4f834586a0bc5373/cluster/get-kube-binaries.sh#L230